### DISCUSSION TITLE
How to update the attributes-view.xml file

### DISCUSSION THREAD

**Discussion by DhruvShahOptinaDx** [nroduit/Weasis#593] [Q&A]
how can I configure the attributes-view.xml file to display the PerFrameFunctionalGroupsSequence`s FrameLabel attribute for a multiframe image?

**Comment by nroduit** [comment]
This is not currently possible. The FRAME level would have to be added, as in the series separation rules.

**Reply by DhruvShahOptinaDx** [reply]
Is there any other way to display label or any kind of information per frame?

For Example: I want to display the wavelength at which each frame was captured
